multigen/pipes.py

```diff
@@ -325,7 +325,8 @@ class Cond2ImPipe(BasePipe):
         "soft-sobel": 0.3,
         "soft": 0.95, #0.5
         "depth": 0.5,
-        "inpaint": 1.0
+        "inpaint": 1.0,
+        "qr": 1.5
```

Comment:
What is qr?

Reply:
yes, it this model https://huggingface.co/DionTimmer/controlnet_qrcode